web/src/components/pop/pop-card-dropdown.tsx

```diff
@@ -17,23 +18,32 @@ interface Props {
   setCopied: (copied: boolean) => void;
   setShowForkPEPModal: (show: boolean) => void;
   setShowRemovePEPModal: (show: boolean) => void;
+  starNumber: number;
 }
 
 export const PopCardDropdown: FC<Props> = (props) => {
   let { namespace, project: name } = useParams();
 
-  const { project, isStarred, copied, setCopied, setShowForkPEPModal, setShowRemovePEPModal } = props;
+  const { project, isStarred, copied, setCopied, setShowForkPEPModal, setShowRemovePEPModal, starNumber } = props;
 
   const { user } = useSession();
 
   const { isPending: isAddingStar, addStar } = useAddStar(user?.login || '');
   const starRemoveMutation = useRemoveStar(user?.login || '');
 
+  const [localStarred, setLocalStarred] = useState(isStarred);
+
+  useEffect(() => {
+    setLocalStarred(isStarred);
+  }, [project]);
+
```

Comment:
Same here with the `<PopCardDropdown />`

`key={isStarred}`

Reply:
Fixed